gradle/bintray.gradle

```diff
@@ -0,0 +1,82 @@
+import java.text.SimpleDateFormat
+
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+apply plugin: 'maven'
+
+if (project.hasProperty("publishRepo") && publishRepo == "bintrayRelease") {
+    bintray {
+        if (!project.hasProperty("publishUsername") || !project.hasProperty("publishPassword")) {
+            throw new GradleException("missing bintray publishing credentials")
+        }
+        user = publishUsername
+        key = publishPassword
+        pkg {
+            repo = "${bintrayRepoName}"
+            name = "${bintrayPackageName}"
+            userOrg = "${bintrayOrgName}"
+            licenses = ['Apache-2.0']
+            publish = "${autoPublish}"
+            vcsUrl = 'https://github.com/pravega/hadoop-connectors.git'
+            version {
```

Comment:
One clarification, is there a relationship between this version here and the one here:

https://github.com/pravega/hadoop-connectors/pull/32/files#diff-c197962302397baf3a4cc36463dce5eaR111

Reply:
no, there is no relationship between them.